framework/include/variables/MooseVariableFV.h

```diff
@@ -714,6 +717,27 @@ MooseVariableFV<OutputType>::evaluate(const ElemArg & elem_arg, const StateArg &
   return getElemValue(elem_arg.elem, state);
 }
 
+template <typename OutputType>
+typename MooseVariableFV<OutputType>::ValueType
+MooseVariableFV<OutputType>::evaluate(const Node & node_arg, const StateArg & state) const
+{
+  const auto & node_to_elem_map = _mesh.nodeToElemMap();
+  const auto & elem_ids = libmesh_map_find(node_to_elem_map, node_arg.node->id());
+  ValueType sum = 0;
+  unsigned short num_values = 0;
+  for (const auto elem_id : elem_ids)
+  {
+    const Elem * const elem = _mesh.queryElemPtr(elem_id);
+    mooseAssert(elem, "We should have this element available");
+    if (!this->hasBlocks(elem->subdomain_id()))
+      continue;
+    const ElemArg elem_arg{elem, /*correct_skewness=*/false};
+    sum += getElemValue(elem_arg, state);
+    ++num_values;
+  }
+  return sum / num_values;
```

Comment:
not volume weighted?
what is our current use for this?

Reply:
I think something like an inverse distance weighting would be more appropriate. The use for this and the biggest driver for this PR is being able to construct a velocity field at nodes for coupling with surface Dirichlet BCs for displacements in ALE simulations where the surface is displacing

Reply:
Sounds good. Whatever we use we should strive to have something that is centered/second order in the default case (seems good enough) and document it

Reply:
For second order accuracy on boundary nodes, the evaluation would probably need to consider the adjacent cell center gradients to extrapolate

Reply:
I think so. those would be considered indirectly if we do an average of face centered values (instead of element-centered values)

Reply:
That is true. Quite a few more operations unfortunately. Maybe weighting of face-based evaluations should be the default for boundary nodes, and element-based should be the default for internal nodes. (Of course then there is the question of inexpensively determining if you're a boundary node)

Reply:
I like that!
For weighting schemes, I am happy with volume-weighting or inverse-distance weighting.
@grmnptr maybe you know what is more common in FV? 
In FE I dont know either!

Reply:
nevermind this isnt for FV :D